### PR TITLE
Change '0 mn' option to 'Never retry' since this is the current behavior

### LINKED ISSUE #8882: Campaign action from custom plugin is not retried even with $event->setResult(false)
[//]: # (READ THIS BEFORE PUBLISHING YOUR ISSUE)

[//]: # (Please post questions, support requests and feature requests/ideas at https://forum.mautic.org.)
[//]: # (Strictly follow the instructions hereunder, otherwise your issue will be closed for lack of information.)
[//]: # (DO NOT report security vulnerabilities here. Send them to security@mautic.com.)

Detail your bug report:

| Q  | A
| ---| ---
| Confirm it is a Bug report |
| Mautic version|2.16.1| 
| PHP version|7.3| 

[//]: # (Feature request? ---> Go to forum please ---> https://forum.mautic.org/c/ideas)
[//]: # (Enhancement? ---> Go to forum please ---> https://forum.mautic.org/c/ideas)
[//]: # (Need support, have questions? ---> Go to forum please ---> https://forum.mautic.org/c/support)

## Description:

Short:
Custom action is not retried even if I use `$event->setResult(false)`. 

Long:
I'm writing a plugin and I need for a custom action to run over and over and over again until it is completed.  I followed the plugin documentation and create a plugin only with what is needed to test. From the docs

> The listener for dispatched event will have the Mautic\CampaignBundle\Event\CampaignExecutionEvent injected. To note that the action was successfully executed, use $event->setResult($result). $result can be a boolean or an array. Setting false will cause the action to be retried. If an array, it will be stored in the campaign event log’s metadata array (useful for displaying information in the contact time-line).
> 
> Use $event->setFailed() to note that an event failed but should not be retried. Failed events do not appear in a contact’s time-line.

### Steps to reproduce:
1. Install the plugin from https://github.com/dhilst/HelloWorldBundle
2. Go to Setup -> Configuration -> Campaign Settings and set _Wait time before retrying a failed action_ to 0mn
3. Create some contacts
4. Create a segment with that contacts
5. Create a campaign with that segment and next the _HelloWorld - Blast off_ action
6. Tail the log in another terminal `tail -F app/logs/mautic_dev-$(date +%Y-%m0-%d).php`
7. At first terminal run `app/console mautic:campaign:update --env=dev && app/console mautic:campaign:trigger --env=dev` (I'm using dev environment to test the plugin)
8. At the terminal running tail is check that the action was executed by presence of `[2020-06-05 13:54:40] mautic.INFO: ==> BLASTOFF 2020-06-05T13:54:40+00:00` message (the date will change). Press some returns to split the output for the next run
9. At first terminal run `app/console mautic:campaign:update --env=dev && app/console mautic:campaign:trigger --env=dev` again
10. At the terminal running tail no `==> BLASTOFF 2020-06-05T13:54:40+00:00` message, meaning that the action was not tried again.

I expect in 10 to get the `==> BLASTOFF 2020-06-05T13:54:40+00:00` messages, but got only 

```
[2020-06-05 13:55:03] mautic.DEBUG: CAMPAIGN: Triggering kickoff events  
[2020-06-05 13:55:03] mautic.DEBUG: CAMPAIGN: Processing the following events: 15  
[2020-06-05 13:55:03] mautic.DEBUG: CAMPAIGN: No events to process  
[2020-06-05 13:55:03] mautic.DEBUG: CAMPAIGN: Triggering scheduled events  
[2020-06-05 13:55:03] mautic.DEBUG: CAMPAIGN: 0 events scheduled to execute.  
[2020-06-05 13:55:03] mautic.DEBUG: CAMPAIGN: No events to process  
[2020-06-05 13:55:03] mautic.DEBUG: CAMPAIGN: Triggering inaction events  
[2020-06-05 13:55:03] mautic.DEBUG: CAMPAIGN: No events to process  
```
 
### Log errors: 

There are no errors being raised, it just don't call the plugin again.

### PR DESCRIPTION
**Pull request by dhilst**
closes #8882 
When use sets '0 mn' in Settings -> Configuration -> Campaign Settings,
failed actions are never retried. Change the label to match the current
behavior.

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | No
| Automated tests included? |No
| Related user documentation PR URL | No
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | #8882 
| BC breaks? | It shouldn't break anything
| Deprecations? | N/A

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

When use sets _Settings -> Configuration -> Campaign Settings -> Wait time before retrying a failed action_ to `0 mn`, the action are never retried. Is possible to find more information [here](https://github.com/mautic/mautic/issues/8882#issuecomment-639772887).

This PR simply changes the label from `0 mn` to `Never retry` which is the current behavior. This should be enough to avoid user confusion as I get confused. 

[//]: # ( As applicable: )

#### Steps to reproduce the bug:

From #8882 

1. Install the plugin from https://github.com/dhilst/HelloWorldBundle
2. Go to Setup -> Configuration -> Campaign Settings and set _Wait time before retrying a failed action_ to 0mn
3. Create some contacts
4. Create a segment with that contacts
5. Create a campaign with that segment and next the _HelloWorld - Blast off_ action
6. Tail the log in another terminal `tail -F app/logs/mautic_dev-$(date +%Y-%m0-%d).php`
7. At first terminal run `app/console mautic:campaign:update --env=dev && app/console mautic:campaign:trigger --env=dev` (I'm using dev environment to test the plugin)
8. At the terminal running tail is check that the action was executed by presence of `[2020-06-05 13:54:40] mautic.INFO: ==> BLASTOFF 2020-06-05T13:54:40+00:00` message (the date will change). Press some returns to split the output for the next run
9. At first terminal run `app/console mautic:campaign:update --env=dev && app/console mautic:campaign:trigger --env=dev` again
10. At the terminal running tail no `==> BLASTOFF 2020-06-05T13:54:40+00:00` message, meaning that the action was not tried again.

#### Steps to test this PR:

1. Just apply the PR and navigate to _Settings -> Configuration -> Campaign Settings_. The label should be visible 

#### List deprecations along with the new alternative:

N/A

#### List backwards compatibility breaks:

N/A

### CODE REVIEW

**Comment by dhilst** [comment]
The changed label 
image

File: app/bundles/CampaignBundle/Form/Type/ConfigType.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label_attr' => ['class' => 'control-label'],
                 'data'       => $options['data']['campaign_time_wait_on_event_false'],
                 'choices'    => [
-                    'null'  => '0 mn',
+                    'null'  => 'Never retry',
```
Comment (kuzmany) [inline app/bundles/CampaignBundle/Form/Type/ConfigType.php]: Thanks for contribution. Make sense for me 👍 
You should use translation: `mautic.core.never`
Also you should rebase it base on 3.x branch, then we can add it to next 3.0.1 release

**Review by kuzmany — CHANGES_REQUESTED** [review]

File: app/bundles/CampaignBundle/Form/Type/ConfigType.php (hunk 1 above)
Comment (dhilst) [reply app/bundles/CampaignBundle/Form/Type/ConfigType.php]: Okay, I will do it, thanks for the advice

**Comment by RCheesley** [comment]
@dhilst could you rebase to trigger the code coverage report please?

**Comment by dhilst** [comment]
Sorry the late reply, for some reason I'm not getting email notification from github

**Comment by dhilst** [comment]
Rebased

**Review by kuzmany — APPROVED** [review]
👍

**Comment by dhilst** [comment]
Hi, I didn't understand the codecov/patch check, why it failed. Do I left anything pending here?

**Review by npracht — APPROVED** [review]
👍

**Pull request merged**